app/controllers/users_controller.rb

```diff
@@ -32,17 +37,6 @@ def show
   end
 
   def user_params
-    params.require(:user).permit(:email, :password, :full_name)
-  end
-
-  private
-
-  def handle_invitation
-    if params[:invitation_token].present?
-      invitation = Invitation.where(token: params[:invitation_token]).first
-      @user.follow(invitation.inviter)
-      invitation.inviter.follow(@user)
-      invitation.update_column(:token, nil)
-    end
+    params.require(:user).permit(:email, :password, :full_name, :stripeToken)
```

Comment:
Why is `stripeToken` among required params. Why do you think it figures under `:user` as well?